services/minespace-web/src/components/dashboard/mine/projectSummaries/ProjectSummariesTable.js

```diff
@@ -54,6 +56,15 @@ export class ProjectSummariesTable extends Component {
       render: (text) => <div title="Last Updated By">{text}</div>,
       sorter: (a, b) => (a.update_user > b.update_user ? -1 : 1),
     },
+    {
+      title: "First Submitted",
+      dataIndex: "create_timestamp",
+      sorter: dateSorter("create_timestamp"),
+      render: (text, record) => {
+        if (record.status_code !== "Draft") return <div title="First Submitted">{text}</div>;
+        return "N/A";
+      },
```

Comment:
I don't think this date will work for "First Submitted" - What happens if the draft was created 2 months ago, and just submitted now? It will show the created_date, not the submitted_date. 

I think we will need to create a new date field and only set it when the status goes from Draft -> Open.

I'm going to be working on the status update ticket now, would you like me to add that logic and come back to this ticket once that is completed?

Reply:
Totally agreed. Thanks for raising this point.

Reply:
We still  have an old field left over from previous migrations called project_summary_date(never deleted the field), perhaps we can re-purpose that?